### DISCUSSION TITLE
sadly I followed the clone and build instructions and it didnt

### DISCUSSION THREAD

**Discussion by daveHitchman** [solokeys/solo#633] [Q&A] [answered]
git clone --recurse-submodules https://github.com/solokeys/solo
cd solo
make all

...
..
ext -DSOLO_VERSION_MAJ=4 -DSOLO_VERSION_MIN=1 -DSOLO_VERSION_PATCH=5 -DSOLO_VERSION=\"4.1.5-8-gfc8d286\" -O2 -fdata-sections -ffunction-sections -fcommon -g -I../ -I./fido2/ -I./pc -I../pc -I./tinycbor/src -DAES256=1  -DSOLO_EXPERIMENTAL=1 -DDEBUG_LEVEL=1 -c -o version.o
crypto.c:37:10: fatal error: sodium/crypto_sign_ed25519.h: No such file or directory
   37 | #include <sodium/crypto_sign_ed25519.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[1]: *** [Makefile:37: crypto.o] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory '/home/dave/git/solokeys_from_web/solo/fido2'
make: *** [Makefile:54: fido2/libsolo.a] Error 2

It appears that there is something missing or easily overlooked in the instructions.

**Comment by enrikb** [comment, marked as answer]
Yes, for a "PC" build, you have to install libsodium before, as briefly mentioned [here](https://github.com/solokeys/solo1#prereqs-1) in the README. How this can be done depends on your system (OS etc.).

**Reply by daveHitchman** [reply]
Fantastic, many thanks, that appears to have done the trick. Missed it totally! I am getting blind in my old age (no further comments required :) )
